js/file/shared/FilePreview.svelte

```diff
@@ -41,12 +38,16 @@
 									? null
 									: file.orig_name || file.name}
 							>
-								Download
+								{display_file_name(file)}
```

Comment:
Thanks @hannahblair appreciate the suggestion. Putting the extension in all caps feels very visually crowded to me, particularly if there's a list of many files. What do you think about instead always including the extension at the end of the filename even if we trim? So it would look like:

* `important_document984023984903...pdf`

Shorter filenames would remain in their original form:

* `requirements.txt`

Reply:
Fixed:
<img width="630" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/cb8a5e6f-8fed-4866-8529-1a23ef9b2d83">